### PR TITLE
Create FX-PI-01_ParisExample.xml

### PR DESCRIPTION
**Pull request by ue71603**
redo of #637

### CODE REVIEW

File: examples/standards/fxc/FX-PI-01_ParisExample.xml [hunk 1]
```diff
@@ -0,0 +1,2798 @@
...
+(No connection possible between metro/RER and bus)
+
+Metro - Metro for 90 minutes hours from validation;
+Metro - RER and RER - RER within Paris, for 90 minutes hours between the first and last validation;
+Bus - Bus (including transfers between the RATP and OPTILE networks), for 90 minutes between the first and last validation;
+Bus - Tram and Tram - Tram, for 90 minutes between the first and last validation
+
+
+Connection times depend on ticket type
+|:::::::::::::::::::::::::::::::|:::::::::::::::::::|:::::::::::::::::::|:::::::::::::::::::|
```
Comment (skinkie) [inline examples/standards/fxc/FX-PI-01_ParisExample.xml:21]: Update tabs/spaces.

Comment (ue71603) [reply examples/standards/fxc/FX-PI-01_ParisExample.xml:21]: fixed. @nick-knowles   pls use space only..... tabs can change between systems

File: examples/standards/fxc/FX-PI-01_ParisExample.xml [hunk 2]
```diff
@@ -0,0 +1,2798 @@
...
+									<StartTariffZoneRef version="any" ref="tsln:zone_4"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_5"/>
+								</DistanceMatrixElement>
+								<!--matrix zone 5 -->
+								<DistanceMatrixElement version="any" id="tsln:noctilien_z5+z5">
+									<Name>Noctilien zone 5 to zone 5</Name>
+									<Distance>3</Distance>
+									<StartTariffZoneRef version="any" ref="tsln:zone_5"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_5"/>
+								</DistanceMatrixElement>
```
Comment (trurlurl) [inline examples/standards/fxc/FX-PI-01_ParisExample.xml:1365]: What about the opposite directions: zone 5 to zone 4, zone 5 to zone 3, etc.? Or should the terms StartTariffZoneRef and EndTariffZoneRef not be taken too literally?

File: examples/standards/fxc/FX-PI-01_ParisExample.xml [hunk 3]
```diff
@@ -0,0 +1,2798 @@
...
+								</GenericParameterAssignment>
+								<GenericParameterAssignment version="1.0" id="idfm:t_plus_single_trip@paper_onboard@access_duration" order="1">
+									<Name>Maximum Duration for trip</Name>
+									<TypeOfAccessRightAssignmentRef versionRef="efp:v1.0" ref="efp:access_duration"/>
+									<TimeIntervalRef version="1.0" ref="idfm:t_plus_single_trip@travel_time_limit@90m"/>
+								</GenericParameterAssignment>
+							</validityParameterAssignments>
+							<distributionAssignments>
+								<DistributionAssignment version="1.0" id="idfm:t_plus_single_trip@paper_onboard" order="1">
+									<Name>Tonboard purchasee</Name>
```
Comment (trurlurl) [inline examples/standards/fxc/FX-PI-01_ParisExample.xml:1518]:
```suggestion
									<Name>Onboard Purchase</Name>
```

File: examples/standards/fxc/FX-PI-01_ParisExample.xml [hunk 4]
```diff
@@ -0,0 +1,2798 @@
...
+						<FareFrameRef version="1.0" ref="idfm:Paris-Example@navigo@Fare_Definitions"/>
+					</prerequisites>
+					<tariffs>
+						<Tariff version="1.0" id="idfm:t_plus_carnet">
+							<Name>t+ carnet of 10 tickets</Name>
+							<Description> </Description>
+							<TariffBasis>flat</TariffBasis>
+							<qualityStructureFactors>
+								<QualityStructureFactor version="1.0" id="idfm:t_plus_carnet@10_units">
+									<Name>30 tickets</Name>
```
Comment (trurlurl) [inline examples/standards/fxc/FX-PI-01_ParisExample.xml:1759]: Why not 10?
```suggestion
									<Name>10 tickets</Name>
```

File: examples/standards/fxc/FX-PI-01_ParisExample.xml [hunk 5]
```diff
@@ -0,0 +1,2798 @@
...
+									<Url>https://www.ratp.fr/en/how-long-are-metro-rer-or-bus-tickets-valid</Url>
+									<Value>30</Value>
```
Comment (trurlurl) [inline examples/standards/fxc/FX-PI-01_ParisExample.xml:1761]:
```suggestion
									<Value>10</Value>
```

File: examples/standards/fxc/FX-PI-01_ParisExample.xml [hunk 6]
```diff
@@ -0,0 +1,2798 @@
...
+								</QualityStructureFactor>
+							</qualityStructureFactors>
+							<fareStructureElements>
+								<FareStructureElement version="1.0" id="idfm:t_plus_carnet@access">
+									<Name>Access rights</Name>
+									<Description>30 t+ single trip tickets</Description>
```
Comment (trurlurl) [inline examples/standards/fxc/FX-PI-01_ParisExample.xml:1767]:
```suggestion
									<Description>10 t+ single trip tickets</Description>
```

File: examples/standards/fxc/FX-PI-01_ParisExample.xml [hunk 7]
```diff
@@ -0,0 +1,2798 @@
...
+								<Description>Negative discount - charge 0.35 EUR for sms delivery</Description>
+								<PriceUnitRef version="1.0" ref="idfm:euro"/>
+								<DiscountAsValue>-0.35</DiscountAsValue>
+								<CanBeCumulative>true</CanBeCumulative>
+							</DiscountingRule>
+						</pricingRules>
+						<roundings>
+							<Rounding version="1.0" id="idfm:round_down">
+								<Name>Round down to nearest 0.05 euros</Name>
+								<RoundingMethod>up</RoundingMethod>
```
Comment (trurlurl) [inline examples/standards/fxc/FX-PI-01_ParisExample.xml:1973]:
```suggestion
								<RoundingMethod>down</RoundingMethod>
```

File: examples/standards/fxc/FX-PI-01_ParisExample.xml [hunk 8]
```diff
@@ -0,0 +1,2798 @@
...
+					<PricingParameterSet version="2.0" id="idfm:t_plus">
+						<pricingRules>
+							<DiscountingRule version="2.0" id="idfm:concessionary_discount">
+								<DiscountAsPercentage>0.5</DiscountAsPercentage>
+							</DiscountingRule>
+						</pricingRules>
+						<roundings>
+							<Rounding version="2.0" id="idfm:round_down">
+								<Name>Round down to nearest 0.05 euros</Name>
+								<RoundingMethod>up</RoundingMethod>
```
Comment (trurlurl) [inline examples/standards/fxc/FX-PI-01_ParisExample.xml:2230]:
```suggestion
								<RoundingMethod>down</RoundingMethod>
```

File: examples/standards/fxc/FX-PI-01_ParisExample.xml [hunk 9]
```diff
@@ -0,0 +1,2798 @@
...
+								<Name>Reduction for carnet</Name>
+								<Description>21% reduction</Description>
+								<DiscountAsPercentage>0.23</DiscountAsPercentage>
+								<CanBeCumulative>true</CanBeCumulative>
+							</DiscountingRule>
+						</pricingRules>
+						<roundings>
+							<Rounding version="3.0" id="idfm:round_down">
+								<Name>Round down to nearest 0.05 euros</Name>
+								<RoundingMethod>up</RoundingMethod>
```
Comment (trurlurl) [inline examples/standards/fxc/FX-PI-01_ParisExample.xml:2514]:
```suggestion
								<RoundingMethod>down</RoundingMethod>
```

**Review by trurlurl — CHANGES_REQUESTED** [review]
Impressive example.

**Comment by Aurige** [comment]
We need to finalise the Basecamp discussion here https://3.basecamp.com/3256016/buckets/2570434/messages/6988278213 before committing it, but this example is definitely one to have here. 
@trurlurl I have integrated your correction that were not already done ... thanks for the review (available in Basecamp for now, for a larger group discussion)

**Comment by ue71603** [comment]
@Aurige is the discussion finalised on basecamp?Ok like that?

**Review by skinkie — APPROVED** [review]

**Review by trurlurl — CHANGES_REQUESTED** [review]
I see several suggestions and comments (all from me) that are still open. Not sure whether the suggestions are all correct, but it is sure that some of the comments point to errors (e.g., duplicate zone combinations).

File: examples/standards/fxc/FX-PI-01_ParisExample.xml (hunk 2 above)
Comment (ue71603) [reply examples/standards/fxc/FX-PI-01_ParisExample.xml:1365]: I think we will merge, when ok. Probably @Aurige will update the example later on anyway.

**Comment by ue71603** [comment]
> I see several suggestions and comments (all from me) that are still open. Not sure whether the suggestions are all correct, but it is sure that some of the comments point to errors (e.g., duplicate zone combinations).

I added them. I think as it works, we merge.When the basecamp discussion is finished, probably a newer version will be added.

**Review by trurlurl — APPROVED** [review]

**Pull request merged**